packages/hooks/src/useCreateNFT.ts

```diff
@@ -231,30 +262,23 @@ export default function useCreateNFT(
           return createLazyMintedAsset.asset.id
         }
 
-        const { createAsset } = await sdk.CreateAsset({
-          input: {
-            standard,
-            creatorAddress: account.toLowerCase(),
-            description,
-            name,
-            traits: traits || null,
-            ...media,
-          },
+        const { createAssetTransaction } = await sdk.CreateAssetTransaction({
+          chainId: chainId,
+          collectionAddress: collectionAddress,
           creatorAddress: account.toLowerCase(),
-          amount: amount ? amount.toString() : '1',
+          metadata: metadata,
+          supply: amount ? amount.toString() : '1',
           royalties: royalties ? Math.round(royalties * 100) : 0,
         })
-        const asset = createAsset?.asset
-        invariant(asset, ErrorMessages.ASSET_CREATION_FAILED)
         invariant(
-          asset.token.__typename === 'ERC721' ||
-            asset.token.__typename === 'ERC1155',
-          ErrorMessages.ASSET_INVALID_STANDARD,
+          createAssetTransaction,
+          ErrorMessages.ASSET_TRANSACTION_CREATION_FAILED,
```

Comment:
Weird to speak about Asset when the function is `createNft`

Reply:
Agree, but this is what we have everywhere, so better to keep that for consistency. We might change all that in some other work (api, error messages, function names...)